README.md

```diff
@@ -1,21 +1,15 @@
-# Landing page
+# THE MET Museum Landing page
 
-> ❗❗ You will complete this task in the topic **Landing**.
-> Now just choose a design and do the parts from the videos
+An adaptive MET Museum page implemented according to figma mockup [https://www.figma.com/file/lSR1m42L9YwzQwzzxKwHpw/THE-MET]
 
-Choose a design for your landing page:
-- [MET landing](https://www.figma.com/file/lSR1m42L9YwzQwzzxKwHpw/THE-MET)
-- [MYBIKE landing](https://www.figma.com/file/NZQAIydtHo5QkINyGLHNcq/BIKE-New-Version?node-id=0%3A1)
-- [B&O](https://www.figma.com/file/DtkQmQ797hk0nI4KfMi2Uq/BOSE-New-Version?type=design&node-id=6817-212&t=ZTV6Gl8NzaWkJ4FK-0)
-- [Nothing](https://www.figma.com/file/DtkQmQ797hk0nI4KfMi2Uq/BOSE-New-Version?type=design&node-id=6802-139&t=L7eKz5YKLN0m5WxR-0)
+To see a demo click here [https://eugenia-andropova.github.io/layout_landing-page/]
```

Comment:
Add this demo link to the PR description 
<img width="910" alt="image" src="https://github.com/user-attachments/assets/342aaea8-6d46-4568-825f-a166c5375185" />